### DISCUSSION TITLE
android.enableR8=true

### DISCUSSION THREAD

**Discussion by Logic2Paradigm** [dlutton/flutter_tts#249] [General]
Does anyone received this error when syncing project:

The option setting 'android.enableR8=true' is deprecated.
It will be removed in version 5.0 of the Android Gradle plugin.
You will no longer be able to disable R8
Affected Modules: flutter_tts

There is no such line in gradle.properties !

and then `Gradle sync failed: Sync failed: reason unknown`

Please help me to solve it

**Comment by dlutton** [comment]
@Logic2Paradigm can you try adding `android.enableD8=true` to your gradle.properties file and see if that fixes your error?  I'll look into updating that in the plugin as well.

**Reply by Logic2Paradigm** [reply]
I didn't figure out what was the problem. I started a new project from scratch and it works now. Probably something was corrupted there.
